### PR TITLE
Feature/fltr 588

### PR DESCRIPTION
**Pull request by werewolfvit**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+## 0.0.1
+
+* TODO: Describe initial release.
```
Comment (akurdyukov) [inline CHANGELOG.md]: Can you write something here pls?

File: LICENSE [hunk 2]
```diff
@@ -0,0 +1 @@
+TODO: Add your license here.
```
Comment (akurdyukov) [inline LICENSE]: Suppose we should use MIT license

File: example/android/app/local.properties [hunk 3]
```diff
@@ -0,0 +1,8 @@
+## This file must *NOT* be checked into Version Control Systems,
```
Comment (akurdyukov) [inline example/android/app/local.properties]: this file should be removed, I guess

File: ios/docusign_flutter.podspec [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+Pod::Spec.new do |s|
+  s.name             = 'docusign_flutter'
+  s.version          = '0.0.1'
+  s.summary          = 'Docusign SDK flutter plugin'
+  s.description      = <<-DESC
+Docusign SDK flutter plugin
+                       DESC
+  s.homepage         = 'http://example.com'
+  s.license          = { :file => '../LICENSE' }
+  s.author           = { 'Your Company' => 'email@example.com' }
```
Comment (akurdyukov) [inline ios/docusign_flutter.podspec]: I guess we should fill this

**Review by akurdyukov — CHANGES_REQUESTED** [review]

File: .idea/.gitignore [hunk 5]
```diff
@@ -0,0 +1,3 @@
+# Default ignored files
```
Comment (vadlit) [inline .idea/.gitignore]: Please add .idea folder in gitignore

File: android/.idea/.gitignore [hunk 6]
```diff
@@ -0,0 +1,3 @@
+# Default ignored files
```
Comment (vadlit) [inline android/.idea/.gitignore]: .idea should be in gitignore

File: android/src/main/kotlin/com/unitedtraders/mobile/docusign_flutter/DocusignFlutterPlugin.kt [hunk 7]
```diff
@@ -0,0 +1,166 @@
...
+
+        override fun onError(@NonNull exception: DSAuthenticationException) {
+          result.error("AuthFailed", null, null)
+        }
+      }
+    )
+  }
+
+  private fun captiveSinging(@NonNull call: MethodCall, @NonNull result: Result) {
+    // TODO необходимо решить проблему с запуском вне Activity, т.к. само создание Intent от нас скрыто - это проблема.
```
Comment (vadlit) [inline android/src/main/kotlin/com/unitedtraders/mobile/docusign_flutter/DocusignFlutterPlugin.kt]: So signing doesn't work right now?

**Review by vadlit — CHANGES_REQUESTED** [review]

File: android/src/main/kotlin/com/unitedtraders/mobile/docusign_flutter/DocusignFlutterPlugin.kt (hunk 7 above)
Comment (werewolfvit) [reply android/src/main/kotlin/com/unitedtraders/mobile/docusign_flutter/DocusignFlutterPlugin.kt]: fixed

File: android/.idea/.gitignore (hunk 6 above)
Comment (werewolfvit) [reply android/.idea/.gitignore]: done

File: .idea/.gitignore (hunk 5 above)
Comment (werewolfvit) [reply .idea/.gitignore]: done

File: ios/docusign_flutter.podspec (hunk 4 above)
Comment (werewolfvit) [reply ios/docusign_flutter.podspec]: done, need check

File: example/android/app/local.properties (hunk 3 above)
Comment (werewolfvit) [reply example/android/app/local.properties]: done

File: LICENSE (hunk 2 above)
Comment (werewolfvit) [reply LICENSE]: done

File: CHANGELOG.md (hunk 1 above)
Comment (werewolfvit) [reply CHANGELOG.md]: done

**Review by vadlit — APPROVED** [review]

File: example/README.md [hunk 8]
```diff
@@ -0,0 +1,16 @@
+# docusign_flutter_example
+
+Demonstrates how to use the docusign_flutter plugin.
+
+## Getting Started
```
Comment (akurdyukov) [inline example/README.md]: Pls delete this if it's not really relevant

**Review by Malarg — APPROVED** [review]

**Pull request merged**